README.md

```diff
@@ -66,9 +66,8 @@ Create a virtualenv into which you can install the module.
 Run the script with the Fedora configuration::
 
     $ ./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
```

Comment:
```suggestion
./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
```

This removes the shebang from the line, which will cause an error in the terminal if pasted in.